packages/bot-web-ui/src/stores/journal-store.js

```diff
@@ -105,6 +119,6 @@ export default class JournalStore {
 
     @action.bound
     clear() {
-        this.unfiltered_messages.clear();
+        this.unfiltered_messages = this.unfiltered_messages.slice(0, 0); // force array update
```

Comment:
```suggestion
       this.unfiltered_messages = this.unfiltered_messages.slice(); // force array update
```

Reply:
I want to clear the array and update it, `.slice(0, 0)` is OK.

Reply:
Specifiying `0, 0` is the same as specifying nothing, there's another call to `slice` in this file which doesn't specify any arguments.